packages/ses/docs/lockdown.md

```diff
@@ -30,6 +30,7 @@ Each option is explained in its own section below.
 | `reporting`                      | `'platform'`     | `'console'` `'none'`                   | where to report warnings ([details](#reporting-options))
 | `unhandledRejectionTrapping`     | `'report'`       | `'none'`                               | handling of finalized unhandled rejections ([details](#unhandledrejectiontrapping-options)) |
 | `evalTaming`                     | `'safeEval'`     | `'unsafeEval'` `'noEval'`              | `eval` and `Function` of the start compartment ([details](#evaltaming-options)) |
+| `hostEvaluators`                 | `'all'`          | `'none'` `'no-direct'`                 | handling of sloppy and indirect eval ([details](#hostevaluators-options)) |
```

Comment:
not true AFAIR - current default is undefined with legacy behavior.

Reply:
`undefined` is not a currently supported `getEnvironmentOption`

https://github.com/endojs/endo/blob/a7954e962f50145be98ea0aca556b83532f684e1/packages/env-options/src/env-options.js#L65

so the current `_legacy` default represents that, not surfaced in docs since it's under-the-hood

(equivalent to `all`, but doesn't error with a strict CSP to make it backwards-compatible)